scripts/lint

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Execute project linters.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        pre-commit run codespell --all-files
+        pre-commit run doc8 --all-files
+        pre-commit run flake8 --all-files
+        pre-commit run isort --all-files
```

Comment:
IMO `isort` should live in `format` only, not `lint`, so it only gets called once by `scripts/test`.